kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -100,6 +100,8 @@
 <script>
 
   const actions = require('../../actions');
+  const coreActions = require('core-actions');
```

Comment:
Due to this: https://github.com/learningequality/kolibri/blob/b81181e2ed653366fb5bff0df6af331ee608b848/kolibri/core/assets/src/core-app/constructor.js#L53

You can reference these as `require('kolibri').constants` or `require('kolibri').coreActions`.